<ui version="4.0" stdsetdef="1" >
  <class>SensorLoggerDlgWidget</class>
  <widget class="QWidget" name="SensorLoggerDlgWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>363</width>
        <height>254</height>
      </rect>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <widget class="QGroupBox" name="fileFrame" >
          <property name="title" >
            <string>File</string>
          </property>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="KUrlRequester" name="m_fileName" />
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="timerFrame" >
          <property name="title" >
            <string comment="@title:group">Timer Interval</string>
          </property>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="KIntNumInput" name="m_timerInterval" >
                <property name="value" >
                  <number>2</number>
                </property>
                <property name="minimum" >
                  <number>1</number>
                </property>
                <property name="maximum" >
                  <number>99</number>
                </property>
                <property name="suffix" >
                  <string> sec</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="GroupBox1" >
          <property name="title" >
            <string>Alarm for Minimum Value</string>
          </property>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="QCheckBox" name="m_lowerLimitActive" >
                <property name="text" >
                  <string>&amp;Enable alarm</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>Enable the minimum value alarm.</string>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="Spacer1" >
                <property name="sizeHint" >
                  <size>
                    <width>16</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QLabel" name="m_lblLowerLimit" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
                <property name="text" >
                  <string>Lower limit:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="KLineEdit" name="m_lowerLimit" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
                <property name="minimumSize" >
                  <size>
                    <width>70</width>
                    <height>0</height>
                  </size>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="GroupBox1_2" >
          <property name="title" >
            <string>Alarm for Maximum Value</string>
          </property>
          <layout class="QHBoxLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item>
              <widget class="QCheckBox" name="m_upperLimitActive" >
                <property name="text" >
                  <string>E&amp;nable alarm</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>Enable the maximum value alarm.</string>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="Spacer1_2" >
                <property name="sizeHint" >
                  <size>
                    <width>16</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item>
              <widget class="QLabel" name="m_lblUpperLimit" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
                <property name="text" >
                  <string>Upper limit:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="KLineEdit" name="m_upperLimit" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
                <property name="minimumSize" >
                  <size>
                    <width>70</width>
                    <height>0</height>
                  </size>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
    <customwidgets>
    <customwidget>
      <class>KIntNumInput</class>
      <header location="global" >knuminput.h</header>
      <container>0</container>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >knuminput.h</include>
    <include location="local" >knuminput.h</include>
  </includes>
  <connections>
    <connection>
      <sender>m_lowerLimitActive</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_lowerLimit</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_upperLimitActive</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_upperLimit</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_lowerLimitActive</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_lblLowerLimit</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_upperLimitActive</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_lblUpperLimit</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
</ui>
